Dockerfile

```diff
@@ -12,21 +12,21 @@ ARG REMOTE_SOURCE_DIR=/remote-source
 ARG REMOTE_SOURCE_SUBDIR=
 ARG DEST_ROOT=/dest-root
 
-ARG TARGETOS
-ARG TARGETARCH
-ARG GO_BUILD_EXTRA_ARGS=
+ARG GO_BUILD_EXTRA_ARGS="-tags strictfipsruntime"
+ARG GO_BUILD_EXTRA_ENV_ARGS="CGO_ENABLED=1 GO111MODULE=on"
```

Comment:
CGO_ENABLED=1 leads to large binaries (though it enables dynamic linking of C libraries).  Why do we have to enable it when we're using ubi images?

Reply:
Yeah, i get your point about binary size but it seems it's required to be FIPS compliant.

[Here](https://github.com/openshift/check-payload/blob/main/internal/validations/validations.go#L103) you can find the check done in the `check-payload` tool used for analyzing the images.